### ISSUE TITLE
Me and Shortcut buttons are not labelled properly for Voice Assistant

### ISSUE THREAD

**Issue by rezzap** [wordpress-mobile/WordPress-Android#12522]
### Expected behavior

When the Me profile icon, or the four shortcut buttons for Stats/Pages/Posts/Media are selected with Voice Assistance on, the proper label would be mentioned. 

### Actual behavior

Voice Assistant reads them as 'unlabelled button' instead of the proper language. 

### Steps to reproduce the behavior

1. Turn on Voice Assistant from the accessibility settings on a Samsung device (I haven't tested this on another device, but expect it would be the same).
2. Open the WordPress app and go to My Site.
3. Click on the Me Profile icon in the top right, voice assistant doesn't recognize this button.
4. Select the Stats Shortcut button (or any of the top four circle icons).
5. Voice Assistant names this 'unlabelled button'. 

##### Tested on Samsung TabA, Android 9, WPAndroid 15.3-rc.2 

User report in 3170317-zen

**Comment by designsimply** [comment]
Tested and confirmed using WPAndroid 15.4-rc-1 on Pixel 3 Android 10. ([39s](https://cloudup.com/cYUMUITMosv))

Tip: you can also turn on TalkBack by pressing both volume keys for 3 seconds ([ref](https://support.google.com/accessibility/android/answer/6007100?hl=en)). 😮

**Comment by thehenrybyrd** [comment]
Tested in WP Android 20.5-rc-2 on Samsung Galaxy S21.
While Stats/Posts/Pages/Media, now as buttons on "Home" tab, are labeled correctly, the "Me" profile icon is not.

Talk Back currently reads the "Me" profile icon as "unlabeled me".
